mpc_webhooks/models/utils.py

```diff
@@ -16,10 +16,44 @@ def _post(url, payload):
         _logger.info("Webhook Timeout[{0}]".format(url))
 
 
+def publish_change(id,model,trigger,dbname=None):
+    try:
+        env_type = None
+        exchange = ""
+        if "staging" in dbname:
```

Comment:
Do you know if we can use another lookup method for determining a staging instance? We sometimes (rare) create staging instances without adding `staging` in the DB name.

Reply:
I was poking around the server and found they set an env variable for just this purpose, so we'll use that instead.